README.md

```diff
@@ -60,7 +60,7 @@ See [docs/API.md](docs/API.md)
 # Holochain Compatibility
 
 This version of `holochain-conductor-api` is currently working with
-- `holochain/holochain` at commit: [c6e501eceae06f381c80ed50ea7bb971f8d1edcc](https://github.com/holochain/holochain/commit/c6e501eceae06f381c80ed50ea7bb971f8d1edcc)
+- `holochain/holochain` at commit: [1eb5b70f64b2188e6ef0c876367838b8a6a2015d](https://github.com/holochain/holochain/commit/1eb5b70f64b2188e6ef0c876367838b8a6a2015d)
 - hdk version 0.0.100 from crates.io
```

Comment:
can we update this too?

Also, it would be useful to add a date of the holochain commit just for reference here

Reply:
Yeap yeap